Content.Server/Light/Components/ToggleableOccluderComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Shared.DeviceLinking;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Light.Components
```

Comment:
```suggestion
namespace Content.Server.Light.Components;
```
file-scoped namespace
remove the curly braces below